examples/Spec.hs

```diff
@@ -15,7 +14,8 @@ import AuctionExample (auctionTest)
 main :: IO ()
 main =
   Hedgehog.defaultMain $
-    take 0 $ -- TODO remove this when tests are complete
+    drop
```

Comment:
I can't find a good way to deffer tests in hedgehog so I just went with this for until the holes are implemented so the test can run.

Reply:
ok